src/app/model/Mission.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Location,
+  TimeWindow,
+  TimeWindowType,
+  MissionInterface,
+  MissionStatus,
+  MissionType,
+  MissionFundedStatus,
+  MissionPayableStatus,
+  MissionDetails,
+} from "./schema";
+import BaseModel from "./BaseModel";
+
+const defaultLocation: Location = {
+  address: "",
+  lat: 0,
+  long: 0,
```

Comment:
does this need to be `lng`? Or did you fix that?

Reply:
yeah, it need to lng